apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx

```diff
@@ -352,19 +352,12 @@ const LayoutSectionFlex = ({
             />
             <ToggleControl
               property="flexWrap"
-              items={[
-                {
-                  isPressed: true,
-                  Icon: WrapIcon,
-                  value: "wrap",
-                },
-                {
-                  isPressed: false,
-                  Icon: NoWrapIcon,
-                  value: "nowrap",
-                },
-              ]}
-              DefaultIcon={WrapIcon}
+              items={["wrap", "nowrap"]}
+              icons={{
+                wrap: WrapIcon,
+                nowrap: NoWrapIcon,
+                default: WrapIcon,
```

Comment:
what's the points of default if first item is default?

Reply:
well just to not have a logic inside the control that decides the first one is default?

Reply:
default is basically only ever needed when advanced value is used just to not have an empty one, maybe using the first would be fine?

Reply:
actually lets kill the default